src/hooks/icons/useGetDashboardIcons.tsx

```diff
@@ -0,0 +1,14 @@
+import { useQuery } from '@tanstack/react-query';
+import { NormalizedIconRepositoryResult } from '../../tools/server/images/abstract-icons-repository';
+
+export const useGetDashboardIcons = () =>
```

Comment:
Why not called it getDashboardIcons ? It's not a hook just a getter function. 
Also shouldn't it be async ?